src/script/lua_api/l_client.cpp

```diff
@@ -169,6 +174,11 @@ int ModApiClient::l_get_node(lua_State *L)
 {
 	// pos
 	v3s16 pos = read_v3s16(L, 1);
+
+	if (getClient(L)->getCSMFlavourLimits() & CSMFlavourLimit::CSM_FL_LOOKUP_NODES) {
+		// TODO
```

Comment:
Why TODO? There is no acceptable usage of node getters, as explained in the issue and elsewhere, this should just return.